_security/index.md

```diff
@@ -27,12 +27,14 @@ The following topics provide a general description of the features that define s
 
 ### Encryption
 
-Encryption protects both data at rest and in transit. At rest, encryption secures sensitive data stored in a cluster. Some examples of stored data include indexes, logs, swap files, automated snapshots, and all data in the application directory. 
+OpenSearch Project encryption can protect data both in transit and at rest. Security specifically, however, is responsible for managing encryption in transit.
```

Comment:
wdyt about retaining the sentence of examples of stored data and moving it to the paragraph below on `Encryption at rest`? 

I like the previous version of the first sentence because prefacing with `OpenSearch Project` makes it sound like OpenSearch provides encryption for data at rest which is what we are trying to avoid. 

Is `Security` with an upper-case S an editorial decision to represent OpenSearch Security? IMO each instance of `Security` should be replaced with `OpenSearch Security`.

Reply:
@cwperks  I get that. I've rewritten this with the suggestions you provided. See if this comes across better. Thanks for the review.